webapp/src/game/game.controller.ts

```diff
@@ -139,4 +143,51 @@ export class GameController {
 
     return sendResponse();
   }
+
+  @Get("/game")
+  findAll() {
```

Comment:
It's confusing in this controller as you mix HTML and JSON routes. Either they must all use content negociation, or you must have a special path for the API routes.